.github/workflows/checkproto.yaml

```diff
@@ -55,7 +55,7 @@ jobs:
       - name: Set up Cocos-AI
         run: |
           # Rename .pb.go files to .pb.go.tmp to prevent conflicts
-          for p in $(ls manager/*.pb.go); do
+          for p in $(ls pkg/manager/*.pb.go); do
```

Comment:
Why is this package called exactly as the `manager` package at the top level? This does not correspond to how we do things in Magistrala, and adds confusion.

@dborovcanin I think this name in `pkg` should be changed to something else, right? Maybe `managersdk` or something else.

Reply:
the files in pkg/manager are the generated proto files. I moved them there since they are imported in prism/computations, manager/api/grpc and manager/manager. manager/api/grpc and manager/manager is also imported in prism/computations. I moved the generated files to /pkg/manager from manager/manager to avoid circular import

Reply:
It does not matter how files are generated, but how the directory is called. Can we please change the naming, because it is confusing?

Reply:
@dborovcanin please help here urgently - decide naming of the directory and help with the code structure for imports

Reply:
@dborovcanin this is the circular import I was referring to https://github.com/ultravioletrs/cocos/pull/87#discussion_r1528391334